src/test/java/gov/healthit/chpl/aqa/stepDefinitions/AnnouncementsSteps.java

```diff
@@ -0,0 +1,113 @@
+package gov.healthit.chpl.aqa.stepDefinitions;
+
+import java.util.List;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import gov.healthit.chpl.aqa.pageObjects.AnnouncementsPage;
+
+/**
+ * Class AnnouncementsSteps definition.
+ */
+public class AnnouncementsSteps extends Base {
+
+    /**
+     * Get user to the Announcements Page.
+     **/
+    @Given("^I navigate to the Announcements Management page$")
+    public void navigateToAnnouncementsPage() {
+        getDriver().get(getUrl() + "#/administration/announcements");
+    }
+
+    /**
+     * Click Add Announcements on the Announcements page.
+     **/
+    @And("^I add an Announcement$")
+    public void addAnnouncement() {
+        JavascriptExecutor jse = (JavascriptExecutor) getDriver();
+        jse.executeScript("arguments[0].scrollIntoView()", AnnouncementsPage.addAnnouncementsButton(getDriver()));
+        AnnouncementsPage.addAnnouncementsButton(getDriver()).click();
+    }
+
+    /**
+     * Clear the "Title/Announcement-Text/StartDate/EndDate" field and enter a value.
+     * @param value to be entered
+     * @param fieldName is the id of the field in the Add Announcements section
```

Comment:
Given that there is both a "name" and an "id" associated with form fields, it would be helpful to future developers to have the name of the variable match whatever part of the form field the variable is talking about: `fieldId`, perhaps, or, to be even closer to what it really is, maybe something like `inputId`